### PR TITLE
Trev8939/update surface placement

### PR DESCRIPTION
**Pull request by TADraeseke**
Hi @eri9600 - Can I ask you to review this PR? It's just a cleanup of an old sample to move away from the deprecated `SurfacePlacement.DRAPED` to `SurfacePlacement.DRAPED_FLAT` and `SurfacePlacement.DRAPED_BILLBOARDED`.

Thanks!

### CODE REVIEW

File: java/surface-placement/README.md [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
 
 ## Relevant API
-* ArcGISScene
-* Camera
+
 * Graphic
 * GraphicsOverlay
 * LayerSceneProperties.SurfacePlacement
 * SceneProperties
-* SceneView
```
Comment (eri9600) [inline java/surface-placement/README.md]: does this still apply since this can only be set on SceneViews?

File: java/surface-placement/README.md [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
 * Surface
 
-#### Tags
+## About the data
+
+The scene launches with a view of northern Snowdonia National Park. Three points are shown hovering with positions defined by each of the different surface placement modes.
+
+## Additional information
+
+This sample uses an elevation service to add elevation/terrain to the scene. Graphics are positioned relative to that surface for the `DrapedBillboarded`, `DrapedFlat`, and `Relative` surface placement modes.
```
Comment (eri9600) [inline java/surface-placement/README.md]: No `Absolute` placement mode?

File: java/surface-placement/README.metadata.json [hunk 3]
```diff
@@ -6,29 +6,27 @@
...
+        "3D",
+        "Absolute",
+        "Altitude",
+        "Draped",
+        "Elevation",
+        "Floating",
+        "Relative",
+        "Scenes",
+        "Sea level",
+	"Surface placement
```
Comment (eri9600) [inline java/surface-placement/README.metadata.json]: spacing seems a bit off

File: java/surface-placement/src/main/java/com/esri/arcgisruntime/sample/surfaceplacement/MainActivity.java [hunk 4]
```diff
@@ -16,7 +16,10 @@
 
 package com.esri.arcgisruntime.sample.surfaceplacement;
 
+import android.graphics.Color;
 import android.os.Bundle;
+import android.widget.ToggleButton;
+
 import androidx.appcompat.app.AppCompatActivity;
```
Comment (eri9600) [inline java/surface-placement/src/main/java/com/esri/arcgisruntime/sample/surfaceplacement/MainActivity.java]: This import should be moved up with the other android dependencies

File: java/surface-placement/src/main/java/com/esri/arcgisruntime/sample/surfaceplacement/MainActivity.java [hunk 5]
```diff
@@ -40,76 +43,93 @@
...
+    // create a red triangle symbol
+    SimpleMarkerSymbol triangleSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbol.Style.TRIANGLE, Color.RED, 10);
 
-    // create a text symbol for each elevation mode
-    TextSymbol drapedText = new TextSymbol(10, "DRAPED", 0xFFFFFFFF, HorizontalAlignment.LEFT,
+    // create the draped flat overlay
+    GraphicsOverlay drapedFlatOverlay = new GraphicsOverlay();
+    drapedFlatOverlay.getSceneProperties().setSurfacePlacement(SurfacePlacement.DRAPED_FLAT);
+    drapedFlatOverlay.getGraphics().add(new Graphic(point, triangleSymbol));
+    TextSymbol drapedFlatText = new TextSymbol(10, "DRAPED FLAT", Color.WHITE, HorizontalAlignment.LEFT,
```
Comment (eri9600) [inline java/surface-placement/src/main/java/com/esri/arcgisruntime/sample/surfaceplacement/MainActivity.java]: I would keep the comment here: `// create a text symbol for elevation mode`

File: java/surface-placement/src/main/java/com/esri/arcgisruntime/sample/surfaceplacement/MainActivity.java [hunk 6]
```diff
@@ -40,76 +43,93 @@
...
         VerticalAlignment.MIDDLE);
-    TextSymbol relativeText = new TextSymbol(10, "RELATIVE", 0xFFFFFFFF, HorizontalAlignment.LEFT,
+    drapedFlatText.setOffsetY(20);
+    drapedFlatOverlay.getGraphics().add(new Graphic(point, drapedFlatText));
+    mSceneView.getGraphicsOverlays().add(drapedFlatOverlay);
+
+    // create the draped flat overlay
```
Comment (eri9600) [inline java/surface-placement/src/main/java/com/esri/arcgisruntime/sample/surfaceplacement/MainActivity.java]: draped **billboarded**

File: java/surface-placement/README.md [hunk 7]
```diff
@@ -1,29 +1,52 @@
...
 ## How it works
-1. Create a GraphicsOverlay.
-2. Set the surface placement mode `GraphicsOverlay.getSceneProperties().setSurfacePlacement(...)`
-	- `DRAPED`, Z value of graphic has no affect and graphic is attached to surface
-	- `RELATIVE`, position graphic using it's Z value plus the elevation of the surface
-	- `ABSOLUTE`, position graphic using only it's Z value
-3. Add graphics to the graphics overlay, GraphicsOverlay.getGraphics.add(Graphic).
-4. Add the graphics overlay to the SceneView, SceneView.getGraphicsOverlays().add(GraphicsOverlay).
+
+1. Create a `GraphicsOverlay` for each placement mode, setting `SceneProperties.setSurfacePlacement`:
```
Comment (eri9600) [inline java/surface-placement/README.md]: should this be `SceneProperties` or `LayerSceneProperties`?

Comment (TADraeseke) [reply java/surface-placement/README.md]: Good catch, thanks!

**Comment by TADraeseke** [comment]
Thanks @eri9600 - Made those fixes!

**Pull request merged**